src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -159,28 +166,25 @@ const PublicationForm = ({
                 <p>or</p>
 
                 <div className={styles.optionsContainer}>
-                  <FormRadioGroup
+                  <RHFFormFieldRadioGroup
```

Comment:
Would suggest we add a type parameter of `FormValues` so that we get completion on the `name` prop. 

Same for the publication field below.